x-pack/plugins/security_solution/public/resolver/view/clickthrough.test.tsx

```diff
@@ -141,23 +142,20 @@ describe('Resolver, when analyzing a tree that has two related events for the or
           graphElements: simulator.testSubject('resolver:graph').length,
           graphLoadingElements: simulator.testSubject('resolver:graph:loading').length,
           graphErrorElements: simulator.testSubject('resolver:graph:error').length,
-          originNode: simulator.processNodeElements({ entityID: entityIDs.origin }).length,
+          originNodeButton: simulator.processNodePrimaryButton(entityIDs.origin).length,
```

Comment:
this test used to check for the presence of a div, now it checks for a button (that the user can see and interact w/)